parts/linux/cloud-init/artifacts/README-COMPONENTS.md

```diff
@@ -107,6 +107,7 @@ Please refer to [components.cue](../../../../schemas/components.cue) for the mos
 	current?: #ReleaseDownloadURI
 }
 #AzureLinuxOSDistro: {
+	"v3.0"?:  #ReleaseDownloadURI
```

Comment:
modified the schema to support v3.0 for explicit Azure Linux v3.

Reply:
Oh could you add those changes in a separate PR? I didn't notice they were there and i force pushed to this branch for clean up, I apologize

Reply:
no worries. I should have let you know first. It's not a lot of changes. I will create a separate PR.